src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStyle = sourceString.split(';');
+  const style = {};
+
+  Object.create(style);
```

Comment:
The use of `Object.create(style)` is unnecessary here. It doesn't affect the `style` object being populated in the `forEach` loop.